src/debuginfo.cpp

```diff
@@ -1094,6 +1046,14 @@ bool jl_dylib_DI_for_fptr(size_t pointer, object::SectionRef *Section, int64_t *
     struct link_map *extra_info;
     dladdr_success = dladdr1((void*)pointer, &dlinfo, (void**)&extra_info, RTLD_DL_LINKMAP) != 0;
 #else
+#ifdef _OS_DARWIN_
```

Comment:
Independent change?

Reply:
Ah, right, this is https://github.com/JuliaLang/julia/pull/43612, which I have on my local branch too – will rebase once that is merged.